README.md

```diff
@@ -28,22 +28,29 @@ default version after a deployment.
 
 - [x] Registration of new worker versions
 - [x] Creation of versioned worker deployment resources
-- [x] Deletion of unreachable worker deployments
+- [x] Deletion of drained worker deployments
 - [x] Manual, Blue/Green, and Progressive rollouts of new worker versions
 - [ ] Autoscaling of worker deployments
 - [ ] Automated rollover to compatible worker versions
 - [ ] Canary analysis of new worker versions
 - [ ] Optional cancellation after timeout for workflows on old versions
 - [ ] Passing `ContinueAsNew` signal to workflows on old versions
 
+## Terminology
+Note that in Temporal, "Worker Deployment" is sometimes referred to as "Deployment", but since the controller makes
+significant references to Kubernetes Deployment resource, within this repository we will stick to these terms:
+- **Worker Deployment Version**: A version of a deployment or service. It can have multiple Workers, but they all run the same build. Sometimes shortened to "version" or "deployment version."
+- **Worker Deployment**: A deployment or service across multiple versions. In a rainbow deploy, a worker deployment can have multiple active deployment versions running at once.
+- **Deployment**: A Kubernetes Deployment resource.
+
 ## Usage
 
 In order to be compatible with this controller, workers need to be configured using these standard environment
 variables:
 
 - `TEMPORAL_HOST_PORT`: The host and port of the Temporal server, e.g. `default.foo.tmprl.cloud:7233`
 - `TEMPORAL_NAMESPACE`: The Temporal namespace to connect to, e.g. `default`
-- `TEMPORAL_DEPLOYMENT_SERIES`: The name of the deployment series. This must be unique to the worker deployment and should not
+- `TEMPORAL_DEPLOYMENT_NAME`: The name of the worker deployment. This must be unique to the worker deployment and should not
   change between versions.
 - `WORKER_BUILD_ID`: The build ID of the worker. This should change with each new worker rollout.
```

Comment:
change this to `WORKER_DEPLOYMENT_VERSION`

Reply:
actually, this should stay as `TEMPORAL_DEPLOYMENT_NAME` (provided by user) since the build id is generated from the worker spec. Later when we provide an option to specify custom build id, then it could make sense to offer `VERSION` too.

Regardless it's a discussion for another PR.